### PR TITLE
Assignment Day1

### PR DESCRIPTION
**Pull request by AbhishekPandey-successive**

### CODE REVIEW

File: react-app/src/components/Day1/one.png
Comment (Rohit0203) [inline react-app/src/components/Day1/one.png:1]: move it to public folder

File: react-app/src/App.test.js
Comment (Rohit0203) [inline react-app/src/App.test.js:1]: create seprate folder for test

**Review by Rohit0203 — COMMENTED** [review]
resolve the comments

File: react-app/src/components/Day1/one.png
Comment (AbhishekPandey-successive) [reply react-app/src/components/Day1/one.png:1]: okay

File: react-app/src/App.test.js
Comment (AbhishekPandey-successive) [reply react-app/src/App.test.js:1]: okay

**Review by Rohit0203 — APPROVED** [review]

**Pull request merged**